bin/netjsonconfig

```diff
@@ -146,11 +144,9 @@ def print_output(output):
     # if file object, get bytes
     if hasattr(output, 'getvalue'):
         output = output.getvalue()
-    # python2: always print
     # python3: if string, just print
```

Comment:
Comment not required anymore.

Reply:
The comment is not present in my local repo, I remved it and pushed it before ! Hmm! but why it's showing in here ?

Reply:
I am talking about "# python3: if string, just print"; It's there, I can guarantee you.
You might have looked into some other file by mistake!
Please check again. Send screenshot if you don't see it! :smile: